.github/instructions/playwright-tests.instructions.md

```
@@ -0,0 +1,170 @@
+---
+# Playwright Test Instructions
+
+These guidelines outline best practices for writing, organizing, and maintaining Playwright tests for LightSpeed projects, with a focus on WordPress themes, blocks, and custom patterns.
+
+---
+
+## Directory & Setup
+
+- Place Playwright tests in a dedicated directory, e.g. `/tests/playwright/` within your project.
+- Install Playwright and its dependencies:
+  ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)